lib/lib-markdown/src/main/java/org/sagebionetworks/markdown/MarkdownDaoImpl.java

```diff
@@ -1,27 +1,33 @@
 package org.sagebionetworks.markdown;
 
+import com.amazonaws.services.lambda.AWSLambda;
+import com.amazonaws.services.lambda.model.AWSLambdaException;
+import com.amazonaws.services.lambda.model.InvokeRequest;
+import com.amazonaws.services.lambda.model.InvokeResult;
 import org.json.JSONException;
 import org.json.JSONObject;
 import org.springframework.beans.factory.annotation.Autowired;
 
+import java.nio.charset.StandardCharsets;
+
 public class MarkdownDaoImpl implements MarkdownDao{
 
 	public static final String MARKDOWN = "markdown";
 	public static final String OUTPUT = "output";
 	public static final String RESULT = "result";
 	public static final String BASE_URL = "baseURL";
+	public static final String FUNCTION_NAME_FMT = "%s-markdownit:prod";
 
 	@Autowired
-	MarkdownClient markdownClient;
-	String synapseBaseUrl;
+	private AWSLambda lambdaClient;
 
-	public String getSynapseBaseUrl() {
-		return synapseBaseUrl;
-	}
+	String synapseBaseUrl;
+	String stack;
 
 	public void setSynapseBaseUrl(String synapseBaseUrl) {
 		this.synapseBaseUrl = synapseBaseUrl;
 	}
+	public void setStack(String stack) { this.stack = stack; }
```

Comment:
If using the service annotation then you can inject the StackConfiguration to get these properties.